extensions/grapher/src/com/google/inject/grapher/DefaultSubgraphCreator.java

```diff
@@ -0,0 +1,42 @@
+package com.google.inject.grapher;
+
+import java.util.Collection;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import com.google.inject.Binding;
+import com.google.inject.spi.DefaultBindingTargetVisitor;
+import com.google.inject.spi.ExposedBinding;
+
+/**
+ * Default subgraph creator.
```

Comment:
this javadoc isn't terribly useful since it's just repeating the class name.  instead, describe what the file does, why it does it, and maybe how it does it.